damus/Views/SetupView.swift

```diff
@@ -57,20 +57,19 @@ struct SetupView: View {
                     }
                     .buttonStyle(GradientButtonStyle())
                     .padding()
-                    
-                    HStack(spacing: 0) {
-                        Text("By continuing you agree to our ")
```

Comment:
We should never break up sentences like this. It makes it impossible to translate the string properly.